### ISSUE TITLE
model diagram is slightly inconsistent with spec

### ISSUE THREAD

**Issue by markmatney** [IIIF/api#1955]
Apologies if I'm not interpreting the model diagram in Section 2 correctly, but it looks to me that the `annotations` arrow on the right side is directed from "Anno." to "Anno. Page". This is inconsistent with the spec, which says that Annotation resources must not have the `annotations` property.

I also noticed an omission that seems fairly easy to add without changing the diagram much (Range may have `start`). I don't have a strong opinion on this, and I understand that diagramming all possible relationships would hurt readability, but I thought I'd flag it.

**Comment by azaroth42** [comment]
Yup, agreed! That arrow should just be deleted. And agree should be easy to add  Range -- start -> Canvas

**Comment by azaroth42** [comment]
Diagram will need drastic rewriting for 4.0, but should take these changes into account at that point.
